diff --git a/helpers/ui.py b/helpers/ui.py
index 5753fa8..71950fa 100644
--- a/helpers/ui.py
+++ b/helpers/ui.py
@@ -329,7 +329,7 @@ class WindowContext(EnumWithAttrs):
             "label": "Options",
             "rows":[
                 RowType.LIST_MODS,
-                RowType.TGL_BRANCH,
+                # RowType.TGL_BRANCH,
                 RowType.TGL_INSTALL,
                 RowType.TGL_STEAM,
                 RowType.TGL_FULLSCREEN,
